tiledb/sm/query/writers/writer_base.cc

```diff
@@ -214,7 +214,11 @@ WriterBase::WriterBase(
   }
 
   optimize_layout_for_1D();
-  check_var_attr_offsets();
+
+  // Remote global order writes may submit offsets > var size due to cache data.
```

Comment:
Hmmm I don't think so. Are you sure this is the case?

Reply:
Nope, not only is this not the case, but I've been passing tests without this change on the REST server for quite a while.. :man_facepalming: Thanks!